src/main/java/com/cloudbees/jenkins/support/impl/JenkinsLogs.java

```diff
@@ -90,7 +90,7 @@ public Iterable<LogRecord> getLogRecords() {
             }
         });
 
-        final File[] julLogFiles = SupportPlugin.getRootDirectory().listFiles(new LogFilenameFilter());
+        final File[] julLogFiles = SupportPlugin.getLogsDirectory().listFiles(new LogFilenameFilter());
```

Comment:
I guess we need some compatibility for existing installations—we would rather not throw out the few days of logs after the upgrade.